controllers/controllers.js

```diff
@@ -0,0 +1,7 @@
+const { selectCategories } = require("../models/models");
+
+exports.getCategories = (req, res, next) => {
+  selectCategories().then((categories) => {
+    res.status(200).send({ categories });
+  });
```

Comment:
how would we tell the client that something has gone wrong if, for example, the db connection fails and this promise rejects?